Linting issue "Parsing error: value.find is not a function"

### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

9.7.0

### What version of `eslint-plugin-svelte` are you using?

2.42.0

### What did you do?

<details>
<summary>After upgrading to latest version of svelte (5.0.0-next.190), eslint shows linting error "Parsing error: value.find is not a function". There was no error with svelte version "5.0.0-next.184". One of the change with svelte 5.0.0-next.190 is that the values passed to components are not to be enclosed in string, and that probably is causing the issue with the eslint-plugin-svelte
</summary>

```
[https://github.com/kunalshah/my-app/blob/main/eslint.config.js](https://github.com/kunalshah/my-app/blob/main/eslint.config.js)
```
</details>

```svelte
Repo: https://github.com/kunalshah/my-app

"npm run lint" to see the issue being reproduced.
```


### What did you expect to happen?

There should not have been any linting errors.

### What actually happened?

Linting error:
  0:0  error  Parsing error: value.find is not a function


### Link to **GitHub Repo** with Minimal Reproducible Example

[https://github.com/kunalshah/my-app](https://github.com/kunalshah/my-app)

or

[https://stackblitz.com/~/github.com/kunalshah/my-app](https://stackblitz.com/~/github.com/kunalshah/my-app)



### Additional comments

_No response_

--- a/.changeset/tender-peaches-thank.md
+++ b/.changeset/tender-peaches-thank.md
@@ -1,0 +1,5 @@
+---
+"eslint-plugin-svelte": minor
+---
+
+fix(deps): update dependency svelte-eslint-parser to ^0.41.0

--- a/packages/eslint-plugin-svelte/package.json
+++ b/packages/eslint-plugin-svelte/package.json
@@ -48,7 +48,7 @@
   },
   "peerDependencies": {
     "eslint": "^7.0.0 || ^8.0.0-0 || ^9.0.0-0",
-    "svelte": "^3.37.0 || ^4.0.0 || ^5.0.0-next.181"
+    "svelte": "^3.37.0 || ^4.0.0 || ^5.0.0-next.191"
   },
   "peerDependenciesMeta": {
     "svelte": {
@@ -66,7 +66,7 @@
     "postcss-safe-parser": "^6.0.0",
     "postcss-selector-parser": "^6.1.0",
     "semver": "^7.6.2",
-    "svelte-eslint-parser": "^0.40.0"
+    "svelte-eslint-parser": "^0.41.0"
   },
   "devDependencies": {
     "@babel/core": "^7.24.7",
@@ -98,7 +98,7 @@
     "sass": "^1.77.5",
     "source-map-js": "^1.2.0",
     "stylus": "^0.63.0",
-    "svelte": "^5.0.0-next.181",
+    "svelte": "^5.0.0-next.191",
     "svelte-i18n": "^4.0.0",
     "type-coverage": "^2.29.0",
     "yaml": "^2.4.5"

